a8cteam51/team51-plugin-scaffold
# Add dependencies scoping

#### Changes proposed in this Pull Request

* Adds a small framework built on top of `humbug/php-scoper` for scoping 3rd-party PHP libraries; this is useful to prevent conflicts with other plugins when including 3rd-party code;
* Adds a GH Action for automatically building and releasing the code; takes into account the new scoping logic;

#### Testing instructions

* Checkout this branch and run `composer run-script packages-install`. That should generate a new `dependencies` folder with the scoped `psr/log` and `psr/container` libraries.
